docs/deployment.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Deployment and CI/CD Process
+---
+
+# Deployment and CI/CD Process
+
+## Overview
+
+DEV relies on GitHub + Travis to continuous deploy to Heroku. If a Pull Request is merged without the "ci skip" in it's title, it will be automatically deployed to production once the test suite passes. The process currently takes about 20 minute to complete and will need a few addtional minute before the change live.
```

Comment:
"minutes to complete"
"minutes before the change"

Reply:
Updated!